web/package.json

```diff
@@ -84,6 +84,8 @@
     "@types/webpack": "~4",
     "@types/webpack-bundle-analyzer": "^3.9.1",
     "@types/webpack-dev-server": "^3.11.3",
+    "@typescript-eslint/eslint-plugin": "^4.22.0",
+    "@typescript-eslint/parser": "^4.22.0",
```

Comment:
Why did you add those here? I think those are already part of the root package.json and should not be needed here.